### PR TITLE
Switches to using DamageChart.ini for... well, the damage chart.

### PR DESCRIPTION
**Pull request by ThislsAUsername**

### CODE REVIEW

**Comment by ThatIsAUsername** [comment]
So, I guess I'm fine with switching to an external .ini file for weapon damage values, but... why?

**Comment by ThislsAUsername** [comment]
My thought is basically that it's
a) slightly easier to manage than hardcoding the table and
b) good practice to learn/remember some I/O stuff before tackling .ini maps

**Comment by ThatIsAUsername** [comment]
Eh, you are either tweaking a number in a .java file or in a .ini file. The second option doesn't require a recompile, but I don't think these values are going to be volatile. Plus, if we redo the combat system at some point, this might go away anyway.

I'm fine either way; you can merge this if you want (just gotta fix yer conflicts first!).

**Comment by ThislsAUsername** [comment]
.....Right, we're doing that. I forgot. Aight, I'm just gonna hardcode it and re-commit/push.

File: src/Units/Weapons/MechRocket.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 public class MechRocket extends WeaponModel {
 
 	public MechRocket() {
-		super(WeaponType.MECHROCKET, 3);
+		super(WeaponType.MECHZOOKA, 3);
```
Comment (ThatIsAUsername) [inline src/Units/Weapons/MechRocket.java]: Is it MECHROCKET or MECHZOOKA? Or are we happy having both?

File: src/Engine/Driver.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 import Terrain.GameMap;
 import UI.InputHandler;
 import UI.MapView;
+import Units.Weapons.Weapon;
```
Comment (ThatIsAUsername) [inline src/Engine/Driver.java]: Does Driver need this import now that it isn't initializing Weapon?

File: src/Units/Weapons/Weapon.java [hunk 3]
```diff
@@ -1,13 +1,42 @@
 package Units.Weapons;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.StringBufferInputStream;
+import java.util.Scanner;
+import java.util.regex.Pattern;
```
Comment (ThatIsAUsername) [inline src/Units/Weapons/Weapon.java]: I don't think we need these File IO imports any more either.

File: src/Units/Weapons/Weapon.java [hunk 4]
```diff
@@ -1,13 +1,42 @@
...
+
 import Units.Unit;
 import Units.UnitModel;
 
 public class Weapon {
+	
+	private static boolean chartSet = false;
```
Comment (ThatIsAUsername) [inline src/Units/Weapons/Weapon.java]: is chartSet actually used for anything?

File: src/Units/Weapons/MechRocket.java (hunk 1 above)
Comment (ThislsAUsername) [inline src/Units/Weapons/MechRocket.java]: MECHZOOKA. I suppose it could be MECHROCKET, though.

File: src/Engine/Driver.java (hunk 2 above)
Comment (ThislsAUsername) [inline src/Engine/Driver.java]: *sigh* I guess I'll go fix that.

File: src/Units/Weapons/Weapon.java (hunk 4 above)
Comment (ThislsAUsername) [inline src/Units/Weapons/Weapon.java]: Nope. Forgot to take it out.

**Pull request merged**